Source/WTF/wtf/WeakRef.h

```diff
@@ -31,9 +31,84 @@
 #include <wtf/ThreadSafeRefCounted.h>
 #include <wtf/Threading.h>
 #include <wtf/TypeCasts.h>
+#include <wtf/TypeTraits.h>
 #include <wtf/WeakPtrImpl.h>
 
 namespace WTF {
+    template<typename Value, typename HashFunctions> class ListHashSet;
```

Comment:
I think we don't normally indent thing inside `namespace`.

Reply:
Will fix once I get it building.